src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -65,6 +67,11 @@ internal sealed partial class SmartRenameViewModel : INotifyPropertyChanged, IDi
     /// </summary>
     public bool IsAutomaticSuggestionsEnabled { get; private set; }
 
+    /// <summary>
+    /// Determines whether smart rename gets semantic context to augment the request for suggested names.
+    /// </summary>
+    public bool IsUsingContext { get; }
```

Comment:
`IsUsingSemanticContext` or `UsesSemanticContext`.  `IsUsingContext` is very vague (and in the context of `c#` sounds like it has something to do with a `using ...` statement :)